.gitignore

```diff
@@ -7,3 +7,20 @@ dist
 /*.prql
 */*.prql
 target
+playground/node_modules
+book/bindings
+book/book
+book/css
+book/examples
+book/FontAwesome
+book/fonts
+book/integrations
+book/language-features
+book/queries
+book/transforms
+book/*.js
+book/*.html
+book/*.css
+book/.nojekyll
+book/*.png
+book/*.svg
```

Comment:
All these artifacts should go into `book/book`, and that's gitignored in `book`. Did it work differently on your system?